package.json

```diff
@@ -52,6 +52,7 @@
     "@commitlint/config-conventional": "^17.1.0",
     "@typescript-eslint/eslint-plugin": "^5.39.0",
     "ci-info": "^3.3.0",
+    "copyfiles": "^2.4.1",
```

Comment:
This is the wrong `package.json` for it. Please runn the install with `npx lerna add -D copyfiles --scope @netlify/build`

Reply:
As this dependency is only needed inside @netlify/build